### PR TITLE
exp/services/recoverysigner: tidy the usage text of the encryption-tink-keyset commands

### PR DESCRIPTION
**Pull request by leighmcculloch**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
* [ ] I've updated any docs ([developer docs](https://www.stellar.org/developers/reference/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [ ] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [ ] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### What
Move the long descriptions in the encryption-tink-keyset commands so that they display when viewing the help for each command, and make other small improvements to the usage text so that they contain all the information needed to use the commands.

### Why
To tidy the usage text and make it easier to digest and use on its own without referring to the README. The top level command usage is easier to skim over as it focuses on what each subcommand does, and then the individual help for each command will discuss how that command works. The URI format is only discussed in the README but it should be discussed in the usage text instead.

### Known limitations

N/A

### CODE REVIEW

File: exp/services/recoverysigner/cmd/keyset_decrypt.go [hunk 1]
```diff
@@ -23,15 +23,15 @@ func (c *KeysetDecryptCommand) Command() *cobra.Command {
...
+			Usage:       "URI for a remote KMS key used to decrypt the Tink keyset (format: aws-kms://arn:aws:kms:<region>:<account-id>:key/<key-id>)",
 			OptType:     types.String,
 			ConfigKey:   &c.EncryptionKMSKeyURI,
 			FlagDefault: "",
 			Required:    true,
 		},
 		{
 			Name:        "encryption-tink-keyset",
-			Usage:       "Tink keyset to decrypt",
+			Usage:       "Tink keyset to decrypt in JSON format",
```
Comment (marcelosalloum) [inline exp/services/recoverysigner/cmd/keyset_decrypt.go:34]: I suggest this change to make it clear it's the decrypted result who is JSON format, and not the Tink keyset.
```suggestion
			Usage:       "Tink keyset to decrypt to JSON format",
```

If the idea was actually to have the Tink keyset in JSON format, I'd suggest:
```suggestion
			Usage:       "Tink keyset in JSON format to be decrypted",
```

File: exp/services/recoverysigner/cmd/keyset_encrypt.go [hunk 2]
```diff
@@ -23,15 +23,15 @@ func (c *KeysetEncryptCommand) Command() *cobra.Command {
...
+			Usage:       "URI for a remote KMS key used to encrypt the Tink keyset (format: aws-kms://arn:aws:kms:<region>:<account-id>:key/<key-id>)",
 			OptType:     types.String,
 			ConfigKey:   &c.EncryptionKMSKeyURI,
 			FlagDefault: "",
 			Required:    true,
 		},
 		{
 			Name:        "encryption-tink-keyset",
-			Usage:       "Tink keyset to encrypt",
+			Usage:       "Tink keyset to encrypt in JSON format",
```
Comment (marcelosalloum) [inline exp/services/recoverysigner/cmd/keyset_encrypt.go:34]: Same here.

Comment (leighmcculloch) [reply exp/services/recoverysigner/cmd/keyset_encrypt.go:34]:
```suggestion
			Usage:       "Tink keyset in JSON format to be encrypted",
```

File: exp/services/recoverysigner/cmd/keyset_rotate.go [hunk 3]
```diff
@@ -26,15 +26,15 @@ func (c *KeysetRotateCommand) Command() *cobra.Command {
...
+			Usage:       "URI for a remote KMS key used to decrypt/encrypt the Tink keyset (format: aws-kms://arn:aws:kms:<region>:<account-id>:key/<key-id>)",
 			OptType:     types.String,
 			ConfigKey:   &c.EncryptionKMSKeyURI,
 			FlagDefault: "",
 			Required:    false,
 		},
 		{
 			Name:        "encryption-tink-keyset",
-			Usage:       "Tink keyset to rotate",
+			Usage:       "Tink keyset to rotate in JSON format",
```
Comment (leighmcculloch) [inline exp/services/recoverysigner/cmd/keyset_rotate.go:37]:
```suggestion
			Usage:       "Tink keyset in JSON format to be rotated",
```

**Pull request merged**